wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights > Posting Activity detail

Fixes #10614 

To test:
- Go to Stats > Insights.
- On Posting Activity, select `View more`.
- Verify the last 12 months is displayed (including the current month).

---

- On initial view, verify the 'footer' only shows the legend:

<img width="458" alt="legend" src="https://user-images.githubusercontent.com/1816888/49758502-5959c600-fc7c-11e8-89f3-2156fbdf329c.png">

---

- Select various days.
- Verify the selected day is orange, and de-selected days return to their original color.

<img width="113" alt="selected_day" src="https://user-images.githubusercontent.com/1816888/49758638-c53c2e80-fc7c-11e8-8893-76d3506fa2de.png">

- Verify the 'footer' now shows the day's data, above the legend:
<img width="461" alt="day_data" src="https://user-images.githubusercontent.com/1816888/49758536-7393a400-fc7c-11e8-8141-a8eaca38c13e.png">
